src/main/java/base/Game.java

```diff
@@ -341,4 +341,18 @@ public RenderHandler getRenderer() {
     public GameMap getGameMap() {
         return gameMap;
     }
+
+    //replace map to default map
+    public void defualtMap(){       
+       logger.info("Default game map loading started");
+
+       loadSpriteSheet();
+       tileService = new TileService(new File(TILE_LIST_PATH), spriteSheet);
+       gameMap = new GameMap(new File(GAME_MAP_PATH), tileService);
+
+       logger.info("Default game map loaded");
+       
+       renderer.adjustCamera(this, player);
```

Comment:
Would be great to make player appear in the center of map. Now player appears on the same spot as in old map, but if old map was bigger than default - it can cause problems, like player will appear outside of map borders.
I would suggest to create method in Player.java like teleportToCenter(), inside calculate X and Y of the center of current map and place player there. 
If that's too much, let me know, I can create separate issue for that and fix it later